lib/dicom_net/association.ex

```diff
@@ -199,4 +232,64 @@ defmodule DicomNet.Association do
 
     response_ds
   end
+
+  defp cfind_response_header(command, presentation_context_id, status) do
+    asci_de = DataSet.fetch!(command, :AffectedSOPClassUID)
+    mid_de = DataSet.fetch!(command, :MessageID)
+
+    response =
+      Dicom.DataSet.from_keyword_list(
+        AffectedSOPClassUID: DataElement.value(asci_de),
+        CommandField: 0x8020,
+        MessageIDBeingRespondedTo: DataElement.value(mid_de),
+        CommandDataSetType: 0x0001, # This field shall be set to the value of 0101H (Null) if no Data Set is present; any other value indicates a Data Set is included in the Message.
+        Status: status # 0x0000 Success, 0xff00 Pending
+      )
+
+    Dicom.BinaryFormat.serialize_command_data_set(response)
+      |> Pdu.new_data_pdu(presentation_context_id) 
+      |> Pdu.serialize()
+  end
+
+  defp cfind_response_data(presentation_context_id, response) do
+    Dicom.BinaryFormat.serialize(response, [endianness: :little, explicit: true])
+      |> Pdu.new_data_pdu(presentation_context_id, 0x02)
```

Comment:
Between here and line 250 there is some asymmetry between the APIs: One time we set the flag here, one time it's set in Pdu via default. Result: Both modules know about some of this PDU type flags.

For me, it seems to be the `Pdu` module's job to know about how exactly these flags are encoded. The public API of Pdu (the one used here) should not force a user to know specific hex codes. I could imagine this to look rather like `Pdu.new_data_pdu(pc_id, last: true/false, type: :command / :data)`. And then Pdu decides on the hex value based on these arguments.